src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -1090,6 +1036,42 @@ ov::runtime::Tensor generate(const
     return ov::test::utils::create_and_fill_tensor(elemType, targetShape, inGenData.range, inGenData.start_from, inGenData.resolution, inGenData.seed);
 }
 
+ov::runtime::Tensor generate(const
+                             std::shared_ptr<ov::op::v8::MatrixNms>& node,
+                             size_t port,
+                             const ov::element::Type& elemType,
+                             const ov::Shape& targetShape) {
+    if (port == 1) {
+        InputGenerateData inGenData(0, 1, 1000, 1);
```

Comment:
Minoir: looks like same code. Possible to be migrated to one function to be reused